src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -45,119 +54,123 @@ public class AccessController : Controller
         {
             T = localizer;
             _applicationManager = applicationManager;
+            _authorizationManager = authorizationManager;
             _identityOptions = identityOptions;
             _openIdService = openIdService;
             _signInManager = signInManager;
             _userManager = userManager;
             _roleManager = roleManager;
         }
 
-        [HttpGet, HttpPost]
-        public async Task<IActionResult> Authorize()
+        [AllowAnonymous, HttpGet, HttpPost, IgnoreAntiforgeryToken]
```

Comment:
The code diff isn't super helpful here, but the `Authorize` action was split into 3 actions for clarity: `Authorize`, `Accept` and `Deny` (exactly like how it used to be in older versions of the module).